test/com/google/javascript/jscomp/CommandLineRunnerTest.java

```diff
@@ -1678,6 +1678,9 @@ private void test(String[] original, String[] compiled,
                     DiagnosticType warning) {
     Compiler compiler = compile(original);
 
+    assertThat(exitCodes).isNotEmpty();
+    assertThat(exitCodes.get(exitCodes.size() - 1)).isEqualTo(0);
```

Comment:
I would think that exitCodes would always have length 1 so then you could just write

`assertThat(exitCodes).containsExactly(0)`

(if I'm remembering the Truth methods right) but I may be totally wrong.